src/main/java/frc/robot/subsystems/TelescopeSubsystem.java

```diff
@@ -36,7 +40,21 @@ public TelescopeSubsystem() {
         armMotor.setSelectedSensorPosition(0);
 
         currentSetpoint = getCurrentPosition();
-        register();
+
+        if (Constants.IOConstants.LOGGING) {
+
+            final ShuffleboardTab telescopeTab = Shuffleboard.getTab("Telescope");
+
+            ShuffleboardLayout positionLayout = telescopeTab.getLayout("Position", BuiltInLayouts.kList);
+
+            positionLayout.addNumber("Current Angle", () -> getCurrentPosition());
```

Comment:
Why is this called angle?

Reply:
Its called `Current Angle` because its the current angle of the rotator from the absolute encoder after the adjustments done from the raw value of the encoder. I just didn't want it to be confused with raw encoder value and make sure that people reading it understand it's been adjusted from the raw value.

Reply:
Isn't this on the telescope though?

Reply:
Oop, you are right, ~~this was definitely not copy and pasted from rotator~~